challenges/objects-arrays.js

```diff
@@ -88,32 +121,33 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 
 */
-const animalNames = [];
-console.log(animalNames);
+const animalNames = zooAnimals.forEach((animal) => console.log(animal.animal_name, animal.scientific_name));
```

Comment:
Why are you console logging here rather than pushing the values into the `animalNames` array that you should have started with. Since for each just runs a function on each item in the original array, it makes more sense to do it that way and push into the empty array.